### PR TITLE
Nodejs update v0.12 => v4.1.1

### PR DESCRIPTION
**Pull request by scottinet**
Removed the profiling service : it uses an old version of nodegrind because the later versions aren't compatible with nodejs 0.12+
And since the nodegrind project hasn't been maintained for almost a year, and since there aren't any other alternative (except for pm2), it has been decided to remove this service completly.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 kuzzle:
-  image: kuzzleio/kuzzle
+  image: kuzzleio/dev
```
Comment (AnthonySendra) [inline docker-compose.yml]: I think it's still `kuzzleio/kuzzle` https://hub.docker.com/r/kuzzleio/kuzzle/~/dockerfile/

File: docker-compose/test.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 kuzzle:
-  image: kuzzleio/kuzzle
+  image: kuzzleio/dev
```
Comment (AnthonySendra) [inline docker-compose/test.yml]: I think it's still `kuzzleio/kuzzle` https://hub.docker.com/r/kuzzleio/kuzzle/~/dockerfile/

File: docker-compose.yml (hunk 1 above)
Comment (scottinet) [inline docker-compose.yml]: Good catch & fixed

**Comment by AnthonySendra** [comment]
LGTM

**Comment by j33f** [comment]
maybe specify the node engine version in package.json ?

**Comment by ballinette** [comment]
+1

**Pull request merged**